File: src/__tests__/golden/fixtures/java-service/src/main/java/com/example/model/Product.java
package com.example.model;

import com.example.model.User;

public class Product {
    private Long id;
    private String name;
    private double price;
    private User owner;

    public Product(String name, double price, User owner) {
        this.name = name;
        this.price = price;
        this.owner = owner;
    }

    public Long getId() { return id; }
    public String getName() { return name; }
    public double getPrice() { return price; }
    public User getOwner() { return owner; }
}

File: src/__tests__/golden/fixtures/rust-web/src/models/product.rs
use super::user::User;

pub struct Product {
    pub id: u64,
    pub name: String,
    pub price: f64,
    pub owner: User,
}

impl Product {
    pub fn new(name: String, price: f64, owner: User) -> Self {
        Product { id: 0, name, price, owner }
    }
}
